modules/post/linux/gather/password_manager.rb

```diff
@@ -0,0 +1,350 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
```

Comment:
What are your thoughts on renaming this file to be a bit more indicative of it being for manage engine? `modules/post/linux/gather/password_manager.rb` sounds quite generic, as though it's targeting linux's password manager

Reply:
I agree, I renamed it to `modules/post/linux/gather/manageengine_password_manager_creds.rb`. I thought since it is already in `modules/post/linux/`, having `linux` in its name was not needed. That said, I'll be happy to add it if you think it should also be part of the filename.